### PR TITLE
[UPDATE] error friendly messages for null and undefined values

### PR DESCRIPTION
**Pull request by windmillcode0**
You may run your i18n files through ai and ai may not be perfect it may introduce undefined or null values this changes gives more human friendly output for the developer to more easily fix the problem

### CODE REVIEW

File: packages/i18n/src/build.ts [hunk 1]
```diff
@@ -166,6 +166,11 @@ function _parseMessagesObject(
       ];
     }
     case 'object':
+      if ([null, undefined].includes(object)) {
+        throw new Error(
+          `While parsing the @wxt-dev/i18n package has found that one of the values in one of your translation files is null or undefined. You may have wanted this value to be transalted from the original please test as necessary`,
+        );
```
Comment (aklinker1) [inline packages/i18n/src/build.ts:172]: Only the source language should be parsed by the module. That means this isn't a problem with your translation service missing a translation, but a formatting problem with a file maintained by a human. I think we should use a simpler error message that indicates that:

```suggestion
        throw new Error(`Messages file should not contain \`${object}\` (found at "${path.join('.')}"`);
```

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Thanks for the PR, can you add a test? 

https://github.com/wxt-dev/wxt/blob/main/packages/i18n/src/__tests__/build.test.ts

```sh
cd packages/18n
pnpm test
```

File: packages/i18n/src/build.ts (hunk 1 above)
Comment (windmillcode0) [reply packages/i18n/src/build.ts]: Added the test and updated the error message

**Review by aklinker1 — APPROVED** [review]

**Comment by aklinker1** [comment]
Will merge next time I'm on github once the checks pass!

**Comment by aklinker1** [comment]
Released in https://github.com/wxt-dev/wxt/releases/tag/i18n-v0.1.1

**Pull request merged**